<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account_dialog</class>
 <widget class="QDialog" name="account_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaitAiAccount</string>
  </property>
  <widget class="QPlainTextEdit" name="te_id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>id</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="te_pw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>passwd</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="te_msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="te_phonenum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>phone number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
